pkg/roachprod/vm/gce/dns.go

```diff
@@ -30,7 +30,7 @@ import (
 const (
 	dnsManagedZone = "roachprod-managed"
 	dnsDomain      = "roachprod-managed.crdb.io"
-	dnsServer      = "ns-cloud-a1.googledomains.com"
+	dnsServer      = "216.239.32.106"
```

Comment:
Have you seen any documentation that this is static? This section makes me question that:

https://cloud.google.com/dns/docs/update-name-servers#find-ip-addresses

If we do decide to follow this path, let's document: 1) the domain name this IP is associated with; 2) the reason for using IP instead of domain name.